### ISSUE TITLE
Add checks for installed nextflow, java, python dependencies in the runtests.sh file

### ISSUE THREAD

**Issue by rbdavid** [EnzymeFunctionInitiative/EST#102]
I think we should add explicit checks for the essential tools that are used in the EFI pipelines before the sample data is downloaded or the testing module scripts are run. This would allow us to catch instances where errors arise due to incomplete installation/preparation of the work environment and clearly separate those errors from any that might arise from the EFI pipelines.

**Comment by nilsoberg** [comment]
@rbdavid Has this been resolved by PR #134?

**Comment by rbdavid** [comment]
Not exactly, but this is pretty low priority, I think. In my mind, a `nextflow -version` and similar for duckdb, cd-hit, blast, etc would be good to include. If an incomplete installation is present, these checks would throw an error causing the `test_env.sh` or `runtests.sh` script (wherever the checks are placed) to fail before any module scripts get run.

The instructions for installation are thorough, but these checks would be actually checking for "correct" installation (whether we check for the specific versions that we require or just check if the software is installed and in the path).
